app/scripts/views/settings/emails.js

```diff
@@ -23,6 +23,8 @@ define(function (require, exports, module) {
   const EMAIL_REFRESH_SELECTOR = 'button.settings-button.email-refresh';
   const EMAIL_REFRESH_DELAYMS = 350;
 
+  const EMAIL_ENABLED_REGEX = /.+@mozilla\.com$|.+@restmail\.net$|.+@softvisioninc\.eu$|.+@softvision\.(com|ro)$/;
```

Comment:
@shane-tomlinson hm the config is our "pipeline" for `env` variables that can be switched without altering the docker image. Currently I think we have to rebuild the whole image. However in the perfect world (future deploys) we might be able just switch some variable and restart the container.